nautobot_bgp_models/migrations/0003_use_upstream_role_part1.py

```diff
@@ -0,0 +1,48 @@
+# pylint: disable=missing-module-docstring,missing-function-docstring,missing-class-docstring,invalid-name
+# Generated by Django 3.2.17 on 2023-02-12 19:21
+
+from django.db import migrations
+import django.db.models.deletion
+import nautobot.extras.models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("nautobot_bgp_models", "0002_viewsets_migration"),
```

Comment:
Needed for safe new install migrations
```suggestion
        ("nautobot_bgp_models", "0002_viewsets_migration"),
        ("extras", "0061_role_and_alter_status"),
```